modules/ROOT/pages/dataweave-types.adoc

```diff
@@ -482,7 +597,7 @@ that accepts a Null type. For example, the
 xref:dw-core-functions-flatten.adoc#flatten2[flatten] function has two function
 signatures, one of which is `flatten(Null): Null`. The signature indicates that
 `flatten` can accept `null` values. The xref:dw-core-functions-plusplus.adoc[++]
-(concatenate) function, which lacks a signature for handling value so the Null
+(concatenate) function, which lacks a signature for handling value so the `Null`
 type, does not accept `null`.
```

Comment:
The sentence structure is not correct:

"The function, which lacks...FOR HANDLING VALUE SO THE `Null` type, does not...."